### PR TITLE
feat: add cli alphabet print

### PR DESCRIPTION
**Pull request by ovsds**

### CODE REVIEW

**Comment by ovsds** [comment]
alphabet_mapper mod naming is crap refactoring in https://github.com/korvin89/asciidize/issues/19

File: src/alphabet_mapper/alphabet.rs [hunk 1]
```diff
@@ -1,7 +1,36 @@
+use core::str::FromStr;
+
 pub struct Alphabet {
     simbols: Vec<char>,
 }
 
+impl FromStr for Alphabet {
+    type Err = ();
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```
Comment (ovsds) [inline src/alphabet_mapper/alphabet.rs]: Any idea how to implement without error in protocol?

File: src/alphabet_mapper/alphabet.rs [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+impl Alphabet {
+    #[must_use]
+    pub fn new(simbols: Vec<char>) -> Alphabet {
+        return Alphabet { simbols };
+    }
+}
+
+#[must_use]
+pub fn get_default() -> Alphabet {
+    return Alphabet::new(vec![
```
Comment (ovsds) [inline src/alphabet_mapper/alphabet.rs]: Would like to replace with from_str, but don't want to unwrap with panic here

File: src/alphabet_mapper/cli.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    mut stdout: impl std::io::Write,
+    mut stderr: impl std::io::Write,
+) {
+    match command {
+        Some(AlphabetCommand::Print { width, alphabet }) => {
+            print(width, alphabet, &mut stdout, &mut stderr);
+        }
+        None => {
+            writeln!(stderr, "No command provided").unwrap();
+            std::process::exit(exitcode::DATAERR);
```
Comment (ovsds) [inline src/alphabet_mapper/cli.rs]: Probably needs to be replaced with result and exit I main

File: tests/cli/alphabet/none.rs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+use assert_cmd::prelude::*;
+use std::process::Command;
+
+#[test]
+fn test_none() -> Result<(), Box<dyn std::error::Error>> {
+    let mut cmd = Command::cargo_bin("asciidize")?;
+    cmd.arg("alphabet");
+
+    cmd.assert().failure().stderr("No command provided\n");
```
Comment (ovsds) [inline tests/cli/alphabet/none.rs]: Check if possible to rewrite with more vertical style

**Review by korvin89 — APPROVED** [review]

**Pull request merged**